src/systems/link.js

```diff
@@ -0,0 +1,16 @@
+var registerSystem = require('../core/system').registerSystem;
+
+/**
+ * Tracked controls system.
+ * It maintains a list with the available tracked controllers
+ */
+module.exports.System = registerSystem('link', {
```

Comment:
It does. I should probably initialize the variable but the currently peeked link is referenced on the system